README.md

```diff
@@ -254,44 +252,44 @@ Apache Airflow version life cycle:
 
 | Version | Current Patch/Minor | State     | First Release | Limited Support | EOL/Terminated |
 |---------|---------------------|-----------|---------------|-----------------|----------------|
-| 2       | 2.1.3               | Supported | Dec 17, 2020  | Dec 2021        | TBD            |
+| 2.0     | 2.1.3               | Supported | Dec 17, 2020  | Dec 31, 2021    | TBD            |
```

Comment:
The version series is 2 not 2.0
In Airflow 1 - a breaking change could be introduced between 1.x to 1.y which is why each version is in a separate row.
In Airflow 2 - we changed to [semver](https://airflow.apache.org/docs/apache-airflow/stable/release-process.html#airflow-s-release-process) so it doesn't require new row per patch version.

Also I'm not sure if the date for limited support is Dec 31. We don't know it yet.